<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatView</class>
 <widget class="QWidget" name="StatView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="file_title">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="dataTab">
       <widget class="QWidget" name="w">
        <attribute name="title">
         <string>数据</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="dataTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <row>
            <property name="text">
             <string>加工</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>搬运</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>检查</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>等待</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>增值</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>非增值</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>时间</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>占比</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PieView" name="pie"/>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PieView</class>
   <extends>QGraphicsView</extends>
   <header>pwh_contrast/PieView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
